content/en/blog/_posts/2019-11-05-kubernetes-with-microk8s.md

```diff
@@ -2,11 +2,11 @@
 title: 'Running Kubernetes locally on Linux with Microk8s'                                                           
 date: 2019-11-05
 ---
-**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi), Product Manager, Canonical
+**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi)
 
-This is article the second in a series about local deployment options on Linux, and it will cover [MicroK8s](https://microk8s.io/)  -the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
+This article the second in a [series](https://kubernetes.io/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/) about local deployment options on Linux, and it covers [MicroK8s](https://microk8s.io/). Microk8s is the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
```

Comment:
@idvoretskyi , I am having trouble parsing line 7. Add a comma and a few more words (change to what you like) to describe what follows. I _think_ the link to the other article can use an internal link.

```
/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/
```


This article, the second in a [series](/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/) about local deployment options on Linux, covers the installation and use of [MicroK8s](https://microk8s.io/).

nit: The shell windows contain a lot of empty lines. Could you shorten the window height? The size of the text output varies.

Reply:
@kbhawkey done on fixing the line 7.

> nit: The shell windows contain a lot of empty lines. Could you shorten the window height? The size of the text output varies.

How critical is this? I've already destroyed the environment where I did a these things (but may recreate if needed).

Reply:
@idvoretskyi , Not critical at all -- merely a suggestion.

Reply:
@kbhawkey great, thanks for pointing out anyway, I'll note that for the future :)